opentelemetry-instrumentation/src/opentelemetry/instrumentation/_semconv.py

```diff
@@ -173,6 +174,7 @@ class _StabilityMode(Enum):
     HTTP_DUP = "http/dup"
     DATABASE = "database"
     DATABASE_DUP = "database/dup"
+    GEN_AI_LATEST = "gen_ai_latest_experimental"
```

Comment:
For consistency with the other modes, would it make sense to also have a `GEN_AI_LATEST_DUP`? Or maybe not because this part of the semconv doesn't really have an "old" vs "new" version?

Reply:
`DUP` is to duplicate and write both old and new versions of attributes. It came from the [TC's transition plan](https://github.com/open-telemetry/opentelemetry-python-contrib/issues/1768#issuecomment-1760372816) for the HTTP attributes stabilization, and for those Python instrumentors there's "old/new/dup" config to support stepwise opt-in for long-time users ([this epic](https://github.com/open-telemetry/opentelemetry-python-contrib/issues/2453)).

That being said I don't know enough about the gen_ai semconv adherence plan! If it's to do this more simple opt-in without duplication because these instrumentors are relatively new, then we don't need `DUP`.